waiter/src/waiter/core.clj

```diff
@@ -921,9 +924,15 @@
    :token->token-metadata (pc/fnk [[:curator kv-store]]
                             (fn token->token-metadata [token]
                               (sd/token->token-metadata kv-store token :error-on-missing false)))
-   :validate-service-description-fn (pc/fnk [[:state service-description-builder]]
-                                      (fn validate-service-description [service-description]
-                                        (sd/validate service-description-builder service-description {})))
+   :validate-service-description-fn (pc/fnk [[:state authenticator service-description-builder]]
+                                      (let [authentication-providers (into #{"disabled" "standard"} (auth/get-authentication-providers authenticator))]
+                                        (fn validate-service-description [service-description]
+                                          (let [authentication (or (get service-description "authentication") "standard")]
```

Comment:
The default should come from `service-description-defaults`

Reply:
do we ever expect (get service-description "authentication") to be nil at this point?

Reply:
No